packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -61,10 +58,10 @@ export class TriggerManager extends Component {
     })
 
     const trigger = await createTrigger(
-      buildKonnectorTriggerAttributes({
+      triggers.buildAttributes({
         konnector,
         account,
-        cron: buildKonnectorCron(konnector)
+        cron: triggers.buildCron(konnector)
```

Comment:
Nit: Since we are on the subject of better names, what do you think of `cron.fromKonnector` ?